src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -53,6 +53,10 @@ object ScalafixPlugin extends AutoPlugin {
         "Make scalafix invocations incremental. On by default."
       )
 
+    // copy of https://github.com/sbt/sbt/blob/e4231ac03903e174bc9975ee00d34064a1d1f373/main/src/main/scala/sbt/Keys.scala#L400
+    // so that it is available on sbt versions older than 1.4.0
+    val bspEnabled = settingKey[Boolean]("")
```

Comment:
Could you move it out of`autoImport` so it does not shadow the real one in `.sbt` files? (maybe the order of the imports would handle that, but just in case, for IDEs "go to definition" for example)